docs/assets/examples/textgrid/v2/main.go

```diff
@@ -88,5 +91,11 @@ func GetMaroto() core.Maroto {
 
 	m.AddRows(text.NewRow(10, "text with hyperlink", props.Text{Hyperlink: &google}))
 
+	subText1 := entity.NewSubText("This is a text", props.SubText{Color: &props.BlueColor})
+	subText2 := entity.NewSubText(" with multiple", props.SubText{Size: 7})
+	subText3 := entity.NewSubText(" styles", props.SubText{Color: &props.RedColor})
```

Comment:
I tried to do it this way, but the provider depends on the subText structure and if I declare the subtext in the Text package, it generates a cyclical dependency.
I think this would be possible if we remove the dependency that the provider has on subtext, but I don't know how to do this in a nice way